clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/security/AppengineCredentials.groovy

```diff
@@ -16,24 +16,29 @@
 
 package com.netflix.spinnaker.clouddriver.appengine.security
 
-import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+import com.google.api.client.http.HttpRequestInitializer
 import com.google.api.client.http.HttpTransport
 import com.google.api.client.json.JsonFactory
 import com.google.api.client.json.jackson2.JacksonFactory
 import com.google.api.services.appengine.v1.Appengine
+import com.google.auth.http.HttpCredentialsAdapter
+import com.google.auth.oauth2.GoogleCredentials
 import com.netflix.spinnaker.clouddriver.googlecommon.security.GoogleCommonCredentials
+import groovy.transform.CompileStatic
 import groovy.transform.TupleConstructor
 
+@CompileStatic
```

Comment:
Looking at what this annotation does, seems that it gets rid of some Groovy dynamic features: https://stackoverflow.com/a/29611007/19136 . This class is small, would it be better to just migrate it to full Java instead? I'm just trying to avoid that in the future somebody adds some groovy dynamic code without noticing (or knowing) what are the effects of `@CompileStatic`.